index.html

```diff
@@ -8,14 +8,50 @@
   <title>Awesome Books</title>
 </head>
 <body>
+  <header class="navigation">
+    <nav class="navbar">
+      <img src="logo.png" alt="logo" class="logo">
+      <ul class="nav-list">
+          <li id="list" class="nav-link"><a href="#">List</a></li>
+          <li id="add" class="nav-link"><a href="#">Add new</a></li>
+          <li id="contact" class="nav-link"><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
Please ensure that you display the current date below the navigation bar according to the requirement. For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install)